src/style.css

```diff
@@ -0,0 +1,129 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
+}
```

Comment:
you should've connected 3 fonts with 3 DIFFERENT font-weights, but with the same font-family